integrations/symfony-bundle.rst

```diff
@@ -342,6 +343,30 @@ The only steps they need is ``require`` one of the adapter implementations in
 their projects ``composer.json`` and instantiating the ``HttplugBundle`` in
 their kernel.
 
+Mock responses in functional tests
+``````````````````````````````````
+To mock responses in your functional tests, proceed as follow:
+
+.. code-block:: yaml
+
+    # config_test.yml
+    httplug:
+        clients:
+            my_awesome_client:
+                factory: 'httplug.factory.mock' # replace factory
+
+And in your tests:
+
+.. code-block:: php
+
+    // SomeWebTestCase.php
+    $client = static::createClient();
+    // $client->disableReboot(); You might uncomment this if your client (BrowserKit) make multiple requests as kernel is rebooted on each request.
```

Comment:
good idea to mention that!

lets say "If your test has the client (BrowserKit) make multiple requests, you need to disable reboot as the kernel is rebooted on each request." - and i would that put into a comment line above the commented out reboot.